applications/search/src/pages/search-page/connected-search-page.jsx

```diff
@@ -45,7 +50,10 @@ const mapDispatchToProps = dispatch => ({
   fetchReferenceDataIfNeeded: () =>
     dispatch(fetchReferenceDataIfNeededAction(REFERENCEDATA_DISTRIBUTIONTYPE)),
   addConcept: item => dispatch(addConceptAction(item)),
-  removeConcept: uri => dispatch(removeConceptAction(uri))
+  removeConcept: uri => dispatch(removeConceptAction(uri)),
+  setDatasetSort: val => dispatch(setDatasetSortAction(val)),
```

Comment:
For bedre lesbarhet kan du kalle 'val' for noe som sier noe om hva slags verdi som kommer inn, f.eks. sortField